src-cpp/rdkafkacpp.h

```diff
@@ -2420,6 +2420,40 @@ class RD_EXPORT KafkaConsumer : public virtual Handle {
    */
   virtual ErrorCode unassign () = 0;
 
+  /**
```

Comment:
New methods must be added to the end of virtual classes as not to break the ABI.

Reply:
oh yeah, should have remembered that.